contracts/CryptoToFiat.sol

```diff
@@ -0,0 +1,143 @@
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract CryptoToFiat {
+    
+    using SafeMath for uint256;
+
+    // Owner of the contract
+    address public owner;
+
+    // Configurable Parameters
+    string public AGIPrice; // Price JSON
+    uint256 public minBalance;
+    uint256 public conversionUpperLimit; // Upper limit on the AGI conversion
+    uint256 public txnLimitInBlocks; // Number of blocks to be wait between Transactions
```

Comment:
They probably should be client specific as different service providers have different money flow.

Reply:
For example we have two service providers: 'Big Famous' and 'Small Unknown'. Both provide AI services. But 'Big Famous' receives 1000 tokens every day when 'Small Unknown' only 10 tokens. They should have different `conversionUpperLimit` values. 500 tokens for 'Big Famous' and 5 for 'Small Unknown'.

Reply:
conversionUpperLimit is not defined per organization or type of the organization as we dont have the details about the Organization in the contract.

I feel we can limit as a constant for all the organizations.

Reply:
I though about limiting per payment address similarly to  `totalConvertedAmt` which is calculated per payment address. It is a question to design so I probably will ask it in design document.

Reply:
@vsbogd Managing per payment address is going to be very difficult task from administrative point of view.
Instead we can do along with the Authorization Signature but again this need to manage offline and might come across the same complexity in managing per payment address/Org GroupId.

We can think of having Some % from the total claimable will be the Upper Limit. We can configure the percentage as the configuration Parameter in the contract. What do you say?

Reply:
@ksridharbabuus , percent seems better to me. Anyway I would propose adding it as separate issue and PR. Will write a comment to the design document.